tests/modules/programs/zsh/history-disable-config.nix

```
@@ -0,0 +1,10 @@
+{
+  programs.zsh = {
+    enable = true;
+    history.enableConfig = false;
+  };
+
+  nmt.script = ''
+    assertFileNotRegex home-files/.zshrc "setopt HIST_FCNTL_LOCK"
```

Presumably we want to check for _all_ `HIST_` options.
```suggestion
    assertFileNotRegex home-files/.zshrc "setopt HIST_.*"
```